README.md

```diff
@@ -51,7 +51,11 @@ The text on the right side of the shield
 The font that should be used for badge text
 
 #### `fontSize` default _11_
-The size of the badge text; values above 17 will be reset to 17 max
+The size of the badge text; min: 8, max: 17
```

Comment:
The way this was previously written that it would be reset was helpful. Maybe word it to that effect.